src/markdown-pages/install-with-kurl/setup-tls-certs.md

```diff
@@ -43,3 +43,24 @@ After the pod has been restarted direct your browser to `http://<ip>:8800` and r
     
 It's best to complete this process as soon as possible to avoid anyone from nefariously uploading TLS certificates.  After this process has completed, the vulnerability will be closed, and uploading new TLS certificates will be disallowed again.  In order to upload new TLS certificates you must repeat the steps above. 
 <br><br><br>
+
+## Registry
+
+The TLS certificate for the [registry add-on](/docs/add-ons/registry) will be renewed automatically at thirty days prior to expiration if the [ekco add-on](/docs/add-ons/ekco) is enabled with version 0.5.0+.
+
+To manually renew the certificate for the registry prior to automatic renewal, re-run the kURL install script.
+
+## Kubernetes Control Plane
+
+The certificates for the Kubernetes control plane will be renewed automatically at thirty days prior to expiration if the [ekco add-on](/docs/add-ons/ekco) is enabled with version 0.5.0+.
+Renewing the Kubernetes control plane certificates will trigger a restart of the Kubernetes API server, which may affect applications running in the cluster.
+To manually renew the certificates for control plane components prior to automatic renewal, run this command on every primary node:
```

Comment:
This is pretty good, but basic. I think we can add more color to it. Like, if you want to avoid all downtime, you could do something like, have an HA setup (at least 3 master nodes), before the automatic cert renewal, remove one master node from the load balancer, renew its certs, then reactivate it in the load balancer. Repeat for the other master nodes.